internal/action/tab.go

```diff
@@ -124,6 +124,17 @@ func (t *TabList) HandleEvent(event tcell.Event) {
 					return
 				}
 			}
+		case tcell.ButtonNone:
+			if t.List[t.Active()].release {
+				// Mouse release received, while already released:
+				// Check for unreleased tabs which are resizing and reset them.
+				for _, tab := range t.List {
+					if !tab.release && tab.resizing != nil {
+						tab.resizing = nil
+						return
+					}
+				}
```

Comment:
It prevents resizing if the active tab changed, but still doesn't prevent it if the screen stopped.

Looks like we should further unify `ReleaseMouse()` which you unified in the next commit (which is nice BTW, thanks), to make it also reset `tab.resizing` for each tab. (And then probably also reuse `ReleaseMouse()` here, instead of adding this custom logic, because why not.)

Reply:
Very good point and adapted. :+1: